src/uu/stdbuf/src/libstdbuf/src/libstdbuf.rs

```diff
@@ -52,7 +54,7 @@ fn set_buffer(stream: *mut FILE, value: &str) {
         res = libc::setvbuf(stream, buffer, mode, size);
     }
     if res != 0 {
-        crash!(res, "error while calling setvbuf!");
+        eprintln!("could not set buffering of {:?} to mode {}", stream, mode);
```

Comment:
Can we do this without using `:?`. What are we printing here? The pointer address? Is that the same as the file?

Reply:
Oh yeah, that's my bad, that should be the file descriptor of that stream! I'm going to wrap that in an `unsafe` block and  use the `fileno()` function from `libc` to get the descriptor and print that instead.